tools/StaticAnalysis/Exceptions/BreakingChangeIssues.csv

```diff
@@ -787,4 +787,49 @@
 "d:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.RedisCache\Microsoft.Azure.Commands.RedisCache.dll","Microsoft.Azure.Commands.RedisCache.SetAzureRedisCache","Set-AzureRmRedisCache","0","1050","The parameter set 'OnlyMaxMemoryPolicy' for cmdlet 'Set-AzureRmRedisCache' has been removed.","Add parameter set 'OnlyMaxMemoryPolicy' back to cmdlet 'Set-AzureRmRedisCache'."
 "d:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.RedisCache\Microsoft.Azure.Commands.RedisCache.dll","Microsoft.Azure.Commands.RedisCache.GetAzureRedisCache","Get-AzureRmRedisCache","0","1050","The parameter set 'GetByResourceGroup' for cmdlet 'Get-AzureRmRedisCache' has been removed.","Add parameter set 'GetByResourceGroup' back to cmdlet 'Get-AzureRmRedisCache'."
 "d:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.RedisCache\Microsoft.Azure.Commands.RedisCache.dll","Microsoft.Azure.Commands.RedisCache.GetAzureRedisCache","Get-AzureRmRedisCache","0","1050","The parameter set 'GetByRedisCache' for cmdlet 'Get-AzureRmRedisCache' has been removed.","Add parameter set 'GetByRedisCache' back to cmdlet 'Get-AzureRmRedisCache'."
-"d:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.RedisCache\Microsoft.Azure.Commands.RedisCache.dll","Microsoft.Azure.Commands.RedisCache.GetAzureRedisCache","Get-AzureRmRedisCache","0","1050","The parameter set 'GetAllInSubscription' for cmdlet 'Get-AzureRmRedisCache' has been removed.","Add parameter set 'GetAllInSubscription' back to cmdlet 'Get-AzureRmRedisCache'."
\ No newline at end of file
+"d:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.RedisCache\Microsoft.Azure.Commands.RedisCache.dll","Microsoft.Azure.Commands.RedisCache.GetAzureRedisCache","Get-AzureRmRedisCache","0","1050","The parameter set 'GetAllInSubscription' for cmdlet 'Get-AzureRmRedisCache' has been removed.","Add parameter set 'GetAllInSubscription' back to cmdlet 'Get-AzureRmRedisCache'."
```

Comment:
You should be able to remove any of these lines that mention position. Some of the "The parameter '[parameter name]' in cmdlet '[cmdlet name] is no longer in the parameter set 'NamespaceAuthorizationRuleSet'" also seem to be unnecessary, please only keep for those that moved from specific parameter sets to all parameter sets.

Reply:
Please resolve.

Reply:
updated the file